argument.go

```diff
@@ -46,7 +46,7 @@ type help struct{}
 //For args with size 1 (Flag,FlagCounter) multiple shorthand in one argument are allowed,
 //so check - returns the number of occurrences.
 //For other args check - returns 1 if occured or 0 in no
-func (o *arg) check(argument string) int {
+func (o *arg) check(argument string) (count int, err error) {
```

Comment:
I prefer not to use this feature of Go and instead to be explicit about which variables are returned. In case of large or complex functions this may be very confusing. I think better keep it simple as `(int, error)`.